DataFormats/ParticleFlowCandidate/interface/PFCandidatePhotonExtra.h

```diff
@@ -61,7 +61,7 @@ namespace reco {
     void addLCorrClusEnergy(float LCorrE);
     const std::vector<float>& LCorrClusEnergy() const {return LocalCorr_;}
 
-    void SetPFPhotonRes(float Res){Res=MVAResolution_;}
+    void SetPFPhotonRes(float Res){MVAResolution_ = Res;}
```

Comment:
Note that MVAResolution_ is never actually used, so this fix has no effect right now. However, it is good to make the fix to make the method work properly in case it is used in the future.